### PR TITLE
authentication built

### PR DESCRIPTION
**Pull request by ciarasugrue**

### CODE REVIEW

File: src/main/java/org/example/db/AuthDAO.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import java.util.UUID;
+import java.util.Date;
+
+public class AuthDAO {
+    private DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    public boolean validLogin(Login login) {
+        try(Connection c = databaseConnector.getConnection()) {
+
+//            ResultSet rs = st.executeQuery("SELECT `password` FROM `user` WHERE username = "
```
Comment (glenn-hamilton) [inline src/main/java/org/example/db/AuthDAO.java]: You don't need this any more

File: untitled/.gitignore [hunk 2]
```diff
@@ -0,0 +1,29 @@
+### IntelliJ IDEA ###
```
Comment (glenn-hamilton) [inline untitled/.gitignore]: Is this file needed?

**Review by glenn-hamilton — APPROVED** [review]

File: src/main/java/org/example/db/AuthDAO.java (hunk 1 above)
Comment (ciarasugrue) [reply src/main/java/org/example/db/AuthDAO.java]: Removed

File: untitled/.gitignore (hunk 2 above)
Comment (ciarasugrue) [reply untitled/.gitignore]: File deleted.
